apps/docs/src/app/examples/checkbox/customicons/customicons.component.ts

```diff
@@ -0,0 +1,19 @@
+import { CheckboxIcons, registerCustomIcons } from '@anglify/components';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+const checkboxIcons: CheckboxIcons = {
+  iconOnState: 'mdi-star-circle',
+  iconOffState: 'mdi-star-circle-outline',
+  iconPack: 'mdi',
+};
+
+@Component({
+  selector: 'app-custom-icons',
+  templateUrl: './customicons.component.html',
+  styleUrls: ['./customicons.component.scss'],
```

Comment:
Add a dash to the file names please. 
```suggestion
  templateUrl: './custom-icons.component.html',
  styleUrls: ['./custom-icons.component.scss'],
```

Reply:
& `custom-icons.component.ts`